pyk4a/pyk4a.py

```diff
@@ -29,13 +29,17 @@ class PyK4A:
     def __init__(self, config=None, device_id=0, thread_safe: bool = True):
         self._device_id = device_id
         self._config = config if (config is not None) else Config()
-        self._thread_safe = thread_safe
+        self._thread_safe = 1 if thread_safe else 0
```

Comment:
let's store integer for avoiding future conversion

Reply:
Why not bool in python code? 
Would allow using self.thread_safe directly (no need for @property and use PyArg_ParseTuple with `p`).

Reply:
Changed